CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+ Contributing
+
+We love pull requests from everyone. Contributing is easy:
+
+* You can report bugs and request features using the [issues page][issues].
+
+[issues]: https://github.com/pydatagn/Community/issues
+
+* You can easily work on the code if you have a GitHub account ([get one][github]):
+
+[github]: https://github.com/join
+
+Fork the project, then clone the repo:
+
+    git clone git@github.com:your-username/Unique-Python.git
+
+Make your change(s).
+
+Push to your fork and [submit a pull request][pr].
+
+[pr]: https://github.com/pydatagn/Community/compare
+
+At this point you're waiting on us. We like to at least comment on pull requests
+as soon as possible. We may suggest some changes or improvements or alternatives.
+
+Some things that will increase the chance that your pull request is accepted:
+
+* Write a [good commit message][commit].
+
+[commit]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
```

Comment:
I think we can add links to other repositories and a brief description about them, along with our website details.

Reply:
Sure, I will link the Organisation's project listing here.